sklearn/impute/tests/test_impute.py

```diff
@@ -1383,3 +1383,30 @@ def test_imputation_order(order, idx_order):
                                random_state=0).fit(X)
         idx = [x.feat_idx for x in trs.imputation_sequence_]
         assert idx == idx_order
+
+
+def test_simple_imputation_inverse_transform():
```

Comment:
Could you parametrize the test with different `missing_values` type.

Reply:
Testing for two `missing_values`, `[-1, np.nan]`